packages/testing/test/chaiA11yAxe.test.js

```diff
@@ -3,76 +3,62 @@ import { fixture, expect, assert, html } from '../index.js';
 describe('chaiA11yAxe', () => {
   describe('Expect', () => {
     it('passes axe accessible tests', async () => {
-      const el = await fixture(
-        html`
-          <button>some light dom</button>
-        `,
-      );
+      const el = await fixture(html`
```

Comment:
isn't this the prettier formatting style?

Reply:
prettier allows for both styles 👍  you will need to roughly format it like this so it keeps the auto format - imho 2 lines less is worth the effort